### PR TITLE
Update Storefront API

### PR DESCRIPTION
**Pull request by brenoc**
Use new endpoints. Make one resource.

### CODE REVIEW

**Comment by brenoc** [comment]
(WIP)

**Comment by brenoc** [comment]
can someone review it, please? @bellacorbo @tamorim 

@dfosco you can work with this branch, please comment if you get any errors

File: src/stores/SearchStore.js [hunk 1]
```diff
@@ -4,19 +4,23 @@ import pluck from 'lodash-compat/collection/pluck';
 
 function getDataFromResources(state, resources) {
   let path = window.location.pathname + window.location.search;
-  let products = resources['products@vtex.storefront-sdk'];
-  let product = resources['product@vtex.storefront-sdk'];
+  let productsAreas = resources['products@vtex'];
+  let productAreas = resources['product@vtex'];
```
Comment (bellacorbo) [inline src/stores/SearchStore.js]: why are there both ```productsAreas``` and ```productArea```? @brenoc

File: storefront/layout.html [hunk 2]
```diff
@@ -40,7 +40,7 @@
   {% script '//npmcdn.com/immutable@3.7.5/dist/immutable.min.js' %}
   {% script '//npmcdn.com/intl@1.0.0/dist/Intl.min.js' %}
   {% script '//npmcdn.com/axios@0.7.0/dist/axios.min.js' %}
-  {% script '//npmcdn.com/history@1.12.5/umd/History.min.js' %}
+  {% script '//npmcdn.com/history@1.15.x/umd/History.min.js' %}
```
Comment (tamorim) [inline storefront/layout.html]: @brenoc I thought we would remove the `history` from the layout and use the one installed as peerDependency on `react-router`

Comment (tamorim) [inline storefront/layout.html]: @brenoc Weird indeed... I'll create an issue so we can investigate this later.

Comment (brenoc) [inline storefront/layout.html]: :+1:

Comment (tamorim) [inline storefront/layout.html]: #59

**Comment by tamorim** [comment]
@brenoc @bellacorbo I have nothing else to comment, I think everything is ok!
Any considerations?

File: src/stores/SearchStore.js (hunk 1 above)
Comment (bellacorbo) [inline src/stores/SearchStore.js]: hmm.. got it! :)

**Comment by bellacorbo** [comment]
For me is ok too. :+1:

**Pull request merged**